test/types/enum/ferguson/enum-overflow.chpl

```diff
@@ -0,0 +1,17 @@
+enum myenum {
```

Comment:
Could you please add a top-level comment to this file explaining whether this is a legal program i.e. it has a well-defined behavior, and why i.e. what language rules talk about "overflow" going from a to b to c.